src/main/java/lily/Ui.java

```diff
@@ -0,0 +1,241 @@
+package lily;
+
+import lily.task.Task;
+
+import java.util.LinkedList;
+import java.util.Scanner;
+
+/**
+ * Deals with all the interactions with the user.
+ * 
+ * @author Hong Yi En, Ian
+ * @version Jan 2022 (AY21/22 Sem 2)
+ */
+public class Ui {
+    private static final String LS = System.lineSeparator();
+    private static final String INDENT = "    ";
+    private static final String DIVIDER = "▼＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝▼" + LS;
+    private static Scanner userInput;
+    private boolean couldLoad;
+
+    /**
+     * Creates a new UI object which handles interactions with the user.
+     * 
+     * @param loadSuccess Whether a save file was loaded.
+     */
+    public Ui(boolean loadSuccess) {
+        userInput = new Scanner(System.in);
+        couldLoad = loadSuccess;
+    }
+
+    /**
+     * Prints the welcome message to the command line.
+     * 
+     * @param tl The task list to be shown to returning users.
+     */
+    public void showWelcome(TaskList tl) {
+        String logo = 
+                LS + "    ██╗     ██╗██╗     ██╗   ██╗" +
+                LS + "    ██║     ██║██║     ╚██╗ ██╔╝" +
+                LS + "    ██║     ██║██║      ╚████╔╝ " +
+                LS + "    ██║     ██║██║       ╚██╔╝  " +
+                LS + "    ███████╗██║███████╗   ██║   " +
+                LS + "    ╚══════╝╚═╝╚══════╝   ╚═╝   " + LS + LS;
+        String welcomeMessage = logo;
+
+        if (couldLoad) {
+            welcomeMessage += INDENT + "sup, welcome back" + LS
+                    + INDENT + "here's your list from the last time" + LS
+                    + LS
+                    + listTasks(tl.getContents()) + LS + LS
+                    + INDENT + "why is it so full of nonsense";
+        } else {
+            showError(INDENT + "oh bother, a new one");
+            welcomeMessage += INDENT + "hey." + LS
+                    + INDENT + "i don't really wanna track your tasks," + LS
+                    + INDENT + "but i guess i have no choice (っ◞‸◟c)" + LS
+                    + INDENT + "need help with something?" + LS
+                    + LS
+                    + INDENT + "Things you can type" + LS
+                    + getCommands();
+        }
```

Comment:
Good use of indentation when using the '+' operator

Reply:
Thanks!